config/foreman.migrations/20250930000666_remove_grub1.rb

```diff
@@ -0,0 +1,7 @@
+if (proxy = answers['foreman_proxy'])
+  root = proxy['tftp_root']
+  if proxy['tftp_dirs']
+    dirs = proxy['tftp_dirs']
+    dirs.delete_if { |dir| dir == "#{root}/grub" }
+  end
```

Comment:
This should also work
```suggestion
  proxy['tftp_dirs']&.delete_if { |dir| dir == "#{proxy['tftp_root']}/grub" }
```